uweb3/pagemaker/__init__.py

```diff
@@ -198,10 +198,14 @@ def parser(self):
             self.persistent.Set(
                 "__parser",
                 templateparser.Parser(
-                    self.options.get("templates", {}).get("path", self.TEMPLATE_DIR)
+                    self.options.get("templates", {}).get("path", self.TEMPLATE_DIR),
+                    symlink_path=self.options.get("templates", {}).get(
+                        "symlink_path", None
+                    ),
                 ),
             )
         parser = self.persistent.Get("__parser")
+        parser.template_dir = self.TEMPLATE_DIR
```

Comment:
is this needed? Its already in the init right?

Reply:
The init defaults to the PageMaker that is passed to the uweb object. This is needed to correct the template path when a PageMaker is called that is not the default pagemaker: https://github.com/underdarknl/uweb3/blob/06a34571edad77ac5ee36a5531746a455d713e6a/uweb3/__init__.py#L193